src/Storage/Psr6CacheStorage.php

```diff
@@ -26,16 +39,15 @@ public function __construct(CacheItemPoolInterface $cachePool)
     public function fetch($key)
     {
         $item = $this->cachePool->getItem($key);
+        $this->lastItem = $item;
 
-        if ($item->isHit()) {
```

Comment:
According to the spec, `get()` return `null` when `isHit()` is false. And `null` will not pass the `instanceof` check. So the call to `isHit()` is not necessary, we can save a method call and use `get()` directly.